src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -140,6 +140,9 @@ public class AddressIndexer : IAddressIndexer
         /// </summary>
         public const int SyncBuffer = 50;
 
+        // Compaction removes info that is used by unity sdk to get UTXOs. Therefore this feature is disabled.
+        public const bool CompactionEnabled = false;
```

Comment:
With compaction off behavior is the same, performance is increased, but hard drive usage is increased as well. 
With compaction on cirrus test it's around 1.1gb. Currently syncing indexer, will know hard drive impact once it's done.

Reply:
`I think this should only be false when the Unity API is being used`

address indexer is used only for unity api as far as I know. 

Also once it was on it can't be turned off because it requires reindexing. 

So in case compaction feature is not cosnt bool then data if it was enabled before should be persisted to the database. 
I think this can be done in a separate PR once we need addr indexer for smth not unity related.